pkg/imagestreamregister/imagestream_register.go

```diff
@@ -0,0 +1,52 @@
+package imagestreamregister
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/openshift/api/image/docker10"
+	"github.com/openshift/api/image/dockerpre012"
+	imagev1API "github.com/openshift/api/image/v1"
+)
+
+var (
+	GroupName     = "image.openshift.io"
+	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1"}
+	schemeBuilder = runtime.NewSchemeBuilder(addKnownTypes, docker10.AddToScheme, dockerpre012.AddToScheme, corev1.AddToScheme)
```

Comment:
I could take the minimalist approach -- and include the minimum amount that runs without error. My initial approach was to do the opposite -- make as few changes from what the standard AddToScheme would do as possible. Since only one line was failing us, I just removed that line, realizing that if that line wasn't there upstream we wouldn't be repeating this code here.